### ISSUE TITLE
GCP: panic: interface conversion: interface {} is nil, not []interface {}

### ISSUE THREAD

**Issue by nie-kita2** [snyk/driftctl#1311]
**Description**

Running driftctl against GCP infrastructure, I get the panic message.

The Go trace:
```
panic: interface conversion: interface {} is nil, not []interface {}

goroutine 1 [running]:
github.com/snyk/driftctl/pkg/middlewares.(*GoogleIAMBindingTransformer).Execute(0xc00025c408, 0xc00012f1b8, 0xc00012f218, 0x1, 0x1)
	/root/project/pkg/middlewares/google_iam_binding_transformer.go:44 +0xa27
github.com/snyk/driftctl/pkg/middlewares.Chain.Execute(0xc0007fc500, 0x29, 0x50, 0xc00012f1b8, 0xc00012f218, 0xc0007fc500, 0x26)
	/root/project/pkg/middlewares/chain_middleware.go:21 +0x24c
github.com/snyk/driftctl/pkg.DriftCTL.Run(0x29aff80, 0xc0008f8a40, 0x29afe20, 0xc00083ca80, 0x29afd60, 0xc0002843a8, 0xc00083caa0, 0x29b03a0, 0xc0005d86f0, 0x29ee0e0, ...)
	/root/project/pkg/driftctl.go:135 +0xf4a
github.com/snyk/driftctl/pkg/cmd.scanRun(0xc000644320, 0x0, 0x0)
	/root/project/pkg/cmd/scan.go:283 +0x9d8
github.com/snyk/driftctl/pkg/cmd.NewScanCmd.func2(0xc0000bcb00, 0xc000114f00, 0x0, 0x6, 0x0, 0x0)
	/root/project/pkg/cmd/scan.go:115 +0x2a
github.com/spf13/cobra.(*Command).execute(0xc0000bcb00, 0xc000114ea0, 0x6, 0x6, 0xc0000bcb00, 0xc000114ea0)
	/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc000349b80, 0x268aa98, 0xc0002c46c0, 0x0)
	/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x375
main.run(0x0)
	/root/project/main.go:66 +0x578
main.main()
	/root/project/main.go:27 +0x25
```

**Environment**
 - OS: Ubuntu
 - driftctl version: 0.18.3
 - terraform version: 1.1.2
 - terraform providers versions: google@4.5.0

**How to reproduce**
The driftctl call:
`driftctl scan --to gcp+tf --from tfstate:///tmp/prd.tfstate --tf-provider-version 4.5.0`

**Comment by eliecharra** [comment]
Thanks a lot for this report, we'll try to reproduce it ASAP and push a fix soon 👍🏻

**Comment by eliecharra** [comment]
@nie-kita2 Would you mind to send us part of your state file ? We wanted to be sure to have fixed the issue and we do not know to reproduce that.

Can you search in your state file if there is `google_storage_bucket_iam_binding` or `google_project_iam_binding` resource with empty `members` attribute ? Can you post theses full resource (after redacting private things) from your state ?

Thanks for your help 🙏🏻

**Comment by nie-kita2** [comment]
@eliecharra, sorry, but I would better avoid sharing the whole state-file. 

However your suggestion nailed down the issue - I indeed had some `google_storage_bucket_iam_binding` resources with "[]" value in the state. Removing them resolved the issue with driftctl panic, now the driftctl scan is running throught as expected.

**Comment by eliecharra** [comment]
I was not asking for the whole state file, just for `google_storage_bucket_iam_binding` and `google_project_iam_binding` with empty `members` keys. Good to know it solve your issue that what I wanted to know. We are working on a fix to make the code more robust about that thanks 👍🏻

**Comment by eliecharra** [comment]
Should be fixed in `v0.18.4`
